test/addDeleteTask.test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @jest-environment jsdom
+ */
+import addTask from '../src/modules/addTask.js';
+import deleteTask from '../src/modules/deleteTask.js';
+import {
+  displayTasks, updateLocalStorage, tasksArray, reindexTasks, displayAllTasks,
+} from '../src/modules/task.js';
+
+jest.mock('../__mock__/testlist.js');
```

Comment:
- [ ] Your tests do not run. Kindly rectify that.

  ![image](https://user-images.githubusercontent.com/54848363/230660032-a447cb18-d5d4-4605-b529-91189e4284ec.png)